examples/chat/chat_test.go

```diff
@@ -43,6 +44,8 @@ func TestMain(t *testing.T) {
 
 		go startPeer(ctx, h1, func(network.Stream) {
 			log.Println("Got a new stream!")
+			// Sleep a bit to let h2 print the logs we're waiting for
+			time.Sleep(500 * time.Millisecond)
```

Comment:
This would be nicer with a proper synchronization mechanism, but I'm fine with merging this for now.